### PR TITLE
fix animation jitter when clicking join channel button

### PR DESCRIPTION
**Pull request by williamboman**
(these gifs shows the issue very poorly)

**Before**
jitter1

**After**
jitter

### CODE REVIEW

File: client/css/style.css [hunk 1]
```diff
@@ -667,7 +667,8 @@ kbd {
 }
 
 #sidebar .lobby .add-channel.opened {
-	transform: rotate(45deg);
+	// translateZ(0) enables hardware acceleration, this is to avoid jittering when animiating
```
Comment (astorije) [inline client/css/style.css]: How on Earth did that pass our CSS linter lol?!
Even my local linter (also stylelint) reported this, and actually whatever error I make in the CSS, `npm run lint:css` doesn't flag. There is something wrong in there, I'ma look right after...

Comment (astorije) [reply client/css/style.css]: `animiating` :)

**Comment by astorije** [comment]
I can't see this issue on my laptop (macOS, Chrome) or phone (iOS, Safari). I'll try with an old phone (Android, Chrome) to see what happens there.

Just curious, what browser are you using? The rotation doesn't seem centered, which is something I'm not seeing either.

File: client/css/style.css (hunk 1 above)
Comment (williamboman) [reply client/css/style.css]: Too used to writing SCSS :), fixing

**Comment by williamboman** [comment]
Fixed comment now

**Pull request merged**